### PR TITLE
[Docs] big cleanup

### PR DESCRIPTION
**Pull request by OskarStark**

### CODE REVIEW

File: docs/reference/filter_field_definition.rst [hunk 1]
```diff
@@ -160,50 +146,36 @@ If you need to filter your base entities by the value of a sub entity property,
...
-    protected function configureDatagridFilters(DatagridMapper $datagrid)
+    protected function configureDatagridFilters(DatagridMapper $datagridMapper)
     {
-        $datagrid
-            // ..
-            ->add('tags', null, array('label' => 'les tags'), null, array('expanded' => true, 'multiple' => true))
-            // ..
-        ;
+        $datagridMapper
+            ->add('tags', null, array('label' => 'les tags'), null, array('expanded' => true, 'multiple' => true));
```
Comment (kunicmarko20) [inline docs/reference/filter_field_definition.rst]: there is still long array syntax here

File: docs/reference/filter_field_definition.rst [hunk 2]
```diff
@@ -213,31 +185,25 @@ To create a custom callback filter, two methods need to be implemented:
...
         {
             $datagridMapper
                 ->add('title')
                 ->add('enabled')
-                ->add('tags', null, array(), null, array('expanded' => true, 'multiple' => true))
+                ->add('tags', null, [], null, ['expanded' => true, 'multiple' => true])
                 ->add('author')
-                ->add('with_open_comments', 'doctrine_orm_callback', array(
+                ->add('with_open_comments', 'doctrine_orm_callback', [
     //                'callback'   => array($this, 'getWithOpenCommentFilter'),
```
Comment (kunicmarko20) [inline docs/reference/filter_field_definition.rst]: there is still long array syntax here

Comment (OskarStark) [reply docs/reference/filter_field_definition.rst]: Thank you 👍🏻

**Comment by OskarStark** [comment]
fixed build

**Pull request merged**